week06_deployment/05_separate_apis/inference_pb2.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: inference.proto
```

Comment:
Do we need to commit this, since it's going to be generated anyway?

Reply:
Fair enough